### PR TITLE
Updating .kproj files

### PR DESCRIPTION
**Pull request by BrennanConroy**
@muratg

### CODE REVIEW

File: src/Microsoft.AspNet.Diagnostics.Elm/Microsoft.AspNet.Diagnostics.Elm.kproj [hunk 1]
```diff
@@ -4,17 +4,14 @@
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
-
   <Import Project="$(VSToolsPath)\AspNet\Microsoft.Web.AspNet.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
     <ProjectGuid>624b0019-956a-4157-b008-270c5b229553</ProjectGuid>
-    <RootNamespace>Microsoft.AspNet.Diagnostics.Elm</RootNamespace>
```
Comment (muratg) [inline src/Microsoft.AspNet.Diagnostics.Elm/Microsoft.AspNet.Diagnostics.Elm.kproj]: Is this removed by design or accidentally?

**Comment by muratg** [comment]
Apart from the comment above, :shipit:

File: src/Microsoft.AspNet.Diagnostics.Elm/Microsoft.AspNet.Diagnostics.Elm.kproj (hunk 1 above)
Comment (BrennanConroy) [inline src/Microsoft.AspNet.Diagnostics.Elm/Microsoft.AspNet.Diagnostics.Elm.kproj]: The script removes it, I'm pretty sure it is not needed.

Comment (muratg) [inline src/Microsoft.AspNet.Diagnostics.Elm/Microsoft.AspNet.Diagnostics.Elm.kproj]: Cool, thanks.

**Pull request closed without merging**